pkgs/development/python-modules/mawk/default.nix

```
@@ -0,0 +1,31 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  poetry-core,
+}:
+
+buildPythonPackage rec {
+  pname = "mawk";
+  version = "0.1.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    repo = "mawk";
+    owner = "jhidding";
+    tag = "v${version}";
+    hash = "sha256-Z/kQ1h+uQWzO4FYJ7Lok9GiXgTHuoQq9icBevPmlVPM=";
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
```

```suggestion
  pythonImportsCheck = [ "mawk" ];

  nativeCheckInputs = [
    pytestCheckHook
  ];
```
To run the test suite